ibis/backends/pandas/execution/selection.py

```diff
@@ -146,8 +147,7 @@ def compute_projection_column_expr(
             name=result_name,
         )
     elif isinstance(expr, ir.DestructColumn):
-        result.columns = expr.type().names
-        return result
+        return coerce_to_dataframe(result, expr.type().names)
```

Comment:
i think this can also be much cleaner if you do the above

Reply:
Updated